src/pages/MemberProfilePage.tsx

```diff
@@ -135,27 +117,34 @@ const Widgets: React.FC<{ member: Member }> = ({ member }) => {
           w="full"
           align="stretch"
         >
-          {member.leetcode && (
+          {member.leetcode && leetcodeStats && (
             <Box flex="1">
               <WidgetCard icon={FaCode} title="LeetCode Stats">
-                <LeetcodeProfile username={member.leetcode.username} />
+                <LeetcodeProfile
+                  username={member.leetcode.username}
+                  stats={{
+                    easy: leetcodeStats['easy_solved'],
+                    medium: leetcodeStats['medium_solved'],
+                    hard: leetcodeStats['hard_solved'],
+                  }}
+                  loading={isLoading}
+                />
               </WidgetCard>
             </Box>
           )}
-          {member.github && (
+          {member.github && githubStats && (
```

Comment:
Same here
```suggestion
          {githubStats && (
```
Test it out though because I'm not 100% sure.